packages/client/src/internals/types.ts

```diff
@@ -18,22 +18,6 @@ export interface AbortControllerInstanceEsque {
   abort(): void;
 }
 
-/**
- * Allows you to communicate with a request and abort it if required via an AbortController.
- */
-export interface AbortSignalEsque {
-  /**
-   * Whether this signal's AbortController has signaled to abort.
-   */
-  readonly aborted: boolean;
-
-  addEventListener<TEvent extends keyof AbortSignalEventMap>(
-    type: TEvent,
-    listener: (this: AbortSignal, ev: AbortSignalEventMap[TEvent]) => any,
-    options?: boolean | AddEventListenerOptions,
-  ): void;
```

Comment:
Maybe we remove this and just keep the 1 other prop 🤷

Reply:
As in remove L30-34

Reply:
That doesn't fix the TypeScript complaint though 😞